_posts/2017-07-11-PyDataDelhi Conference 2017.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  PyData Delhi Conference 2017 : An introduction
+date:   2017-07-11
+categories: python pydata conference
+---
+
+**Date**: 11 July 2017
+
+With the **first ever PyData Conference for Delhi** coming up on **2nd** and **3rd September** at [**IIIT-Delhi**](https://iiitd.ac.in/) , we at **PyData Delhi** are super busy with prep work! Also, needless to say all of us organisers are bubbling with enthusiasm and rushing around, covering all factors and ensuring that everything is perfect for our attendees :D
+
+PyData has collaborated with [**Women Who Code Delhi**](https://www.womenwhocode.com/delhi)  and there has been a lot of creative exchange in terms of ideas and novel design concepts B-)
+All hands aboard! The PyData Conference ship is officially sailing and has begun charting its course. We are all geared up for an eventful and awesome journey till September, where all this hardwork will culminate in bringing tech enthusiasts, students, mentors, speakers together under one roof: to learn , collaborate, discuss, ideate and most of all, enjoy coding!
+
+##### We at **PyData**  and have one mission as we organise and hold conferences: 
+
+  * To provide a *forum* for the international community of users and developers of data analysis tools to *share ideas* and *learn from each other.*   
+   * The global PyData network promotes *discussion of best practices, new approaches, and emerging technologies for data management, processing, analytics, and visualization*. 
+  * PyData communities approach data science using many languages, including (but not limited to) Python, Julia, and R.
+
+##### And **Women Who Code** is a global organisation that aims to:
+  * Envision a world where women are proportionally represented as technical leaders, executives, founders, VCs, board members, and software engineers.
+  * *Inspire* women to excel in technology careers.
+  * *Empower* with skills needed for professional achievements.
+  * *Educate* companies to better promote, retain and hire talented women.
+  * *Build a global community* where networking and mentorship is valued.
+  * *Develop role models* and *support this generation* of engineers.
+
+
+
+And this time, PyData Delhi chapter is proud to hold it's very first conference at IIIT-Delhi. Woot woot! Looking forward to seeing young bright minds and seasoned programmers alike, with us on 2nd and 3rd of September!
+
+*PyData is an educational program of NumFOCUS, a 501(c)3 non-profit organization in the United States.*
+![PyData Delhi](/home/zoha/Desktop/logo.png  "PyData Delhi")
+*Women Who Code is a 501(c)(3) not-for-profit organization*
+![Women Who Code, Delhi](/home/zoha/Desktop/wwc/WWC.png  "Women Who Code, Delhi")
+
+- Zoha Hamid, Member, PyData Delhi; Key Team member, WWC Delhi
```

Comment:
Can we change this to something like:

> written by @zohamid 
on behalf of PyData Delhi & WWC Delhi Team